app/rww/ldp/WebClient.scala

```diff
@@ -137,11 +134,11 @@ class WSClient[Rdf <: RDF](readerSelector: ReaderSelector[Rdf, Try], rdfWriter:
   }
 
   private val AllRdfMimeTypes: Set[MimeType] = Set(
-     MimeType("text","turtle"),
-    MimeType("application","rdf+xml"),
-    MimeType("text","n3")
-  )
-
+    MimeType("text", "turtle"),
+    MimeType("application", "rdf+xml"),
+    MimeType("text", "rdf+xml"),
```

Comment:
I think AllRdfMimeTypes should contain only the official mime types. `text/rdf+xml` is an old no longer recommended one. On the other hand one could add ntriples now, as we have specialised parsers for those.